policyengine_api/routes/household_routes.py

```diff
@@ -0,0 +1,199 @@
+from flask import Blueprint, Response, request
+import json
+
+from policyengine_api.constants import COUNTRY_PACKAGE_VERSIONS
+from policyengine_api.services.household_service import HouseholdService
+from policyengine_api.utils import hash_object
+from policyengine_api.utils.payload_validators import (
+    validate_household_payload,
+    validate_country,
+)
+
+
+household_bp = Blueprint("household", __name__)
+household_service = HouseholdService()
+
+
+@household_bp.route("/<household_id>", methods=["GET"])
+@validate_country
+def get_household(country_id: str, household_id: str) -> Response:
+    """
+    Get a household's input data with a given ID.
+
+    Args:
+        country_id (str): The country ID.
+        household_id (str): The household ID.
+    """
+    print(f"Got request for household {household_id} in country {country_id}")
+
+    # Ensure that household ID is a number
+    try:
+        household_id = int(household_id)
+    except ValueError:
+        return Response(
+            status=400,
+            response=f"Invalid household ID; household ID must be a number",
+        )
+
+    try:
+        household: dict | None = household_service.get_household(
+            country_id, household_id
+        )
+        print(household)
+        if household is None:
+            return Response(
+                json.dumps(
+                    {
+                        "status": "error",
+                        "message": f"Household #{household_id} not found.",
+                    }
+                ),
+                status=404,
+            )
+        else:
+            return Response(
+                json.dumps(
```

Comment:
I don't think we've ever had a standard here - definitely time for one. Let me work up what I think makes sense, gather your feedback, then implement.

Reply:
I'm going to delay migrating the `economy_route` and `economy_service` into the correct structure because I think it will require some testing, and this service is particularly annoying to test; also, it's our core business logic, and there are some other conventions that it lacks that I enforced later (e.g., as part of this PR) that I didn't at the time. I did move the metadata endpoint over.